sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/db/pending/PendingTransactionEntity.kt

```diff
@@ -34,9 +38,13 @@ data class PendingTransactionEntity(
     fun toPendingTransaction(zcashNetwork: ZcashNetwork) = PendingTransaction(
         id = id,
         value = Zatoshi(value),
+        fee = Zatoshi(fee),
         memo = memo?.let { FirstClassByteArray(it) },
         raw = FirstClassByteArray(raw),
-        toAddress = toAddress,
+        recipient = TransactionRecipient.new(
```

Comment:
Instead of having this TransactionRecipient constructor, I think that the correct thing to do is perform the internal checking for null inline here, and return whichever TransactionRecipient variant is correct. Basically do the inverse of what is done down in `from` below.

Reply:
I kept the method because it had another use case inside PendingTransactionEntity, but I moved it to be a private extension function.